backend/windmill-worker/src/worker.rs

```diff
@@ -1122,9 +1121,17 @@ async fn handle_deno_job(
     inner_content: &String,
     timeout: i32,
     shared_mount: &str,
+    lockfile: Option<String>,
 ) -> error::Result<serde_json::Value> {
     logs.push_str("\n\n--- DENO CODE EXECUTION ---\n");
     set_logs(logs, job.id, db).await;
+    let lockfile = lockfile
+        .and_then(|e| if e.starts_with("{") { Some(e) } else { None })
+        .unwrap_or("{}".to_string());
```

Comment:
unwrap_or_else for string allocation (I know the codebase is riddled with me not doing it consistently :))